server/repositories/tests.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     let queries = [];
 
     tests.forEach(test => {
+      // FIXME
```

Comment:
here's where I hit a wall. I think this logic needs to be revisited. 

1. `test.title` and `test.code` will always exist because the schema requires them which means you can't delete a test case
2. I kept seeing a new `pageID` be passed but since a test already existed (has `test.id`) the query was always an update for a record that didn't exist (`UPDATE tests SET ... WHERE pageID = ++pageID AND testID = test.id`)